octoprint_discordremote/command.py

```diff
@@ -89,7 +181,7 @@ def list_files(self):
             description = ''
             title = ''
             try:
-                title = details['path']
+                title = (details['path'])[1:]
```

Comment:
@OctopusProteins Would prefer details['path'][1:], no need for parenthesis

Reply:
Fixed by 5e83f84